src/main/java/org/generationcp/middleware/pojos/workbench/WorkbenchUser.java

```diff
@@ -66,21 +65,89 @@
 
 	private static final long serialVersionUID = 1L;
 
+	public static final String GET_USERS_BY_PROJECT_ID = "SELECT  "
+		+ "  users.userid, "
+		+ "  users.instalid, "
+		+ "  users.ustatus, "
+		+ "  users.uaccess, "
+		+ "  users.utype, "
+		+ "  users.uname, "
+		+ "  users.upswd, "
+		+ "  users.personid, "
+		+ "  users.adate, "
+		+ "  users.cdate, "
+		+ "  pr.fname, "
+		+ "  pr.lname "
+		+ "    FROM "
+		+ "       workbench_project p "
+		+ "           INNER JOIN "
```

Comment:
I believe you can save a join to `workbench_crop `table here by joining `workbench_project.crop_type` to `crop_persons.crop_name`